[K9VULN-15241] Update docs on suppressed violations (#36973)
DataDog/documentation: content/en/security/code_security/static_analysis/configuration/_index.md

Reviewer: ```suggestion
[1]: https://app.datadoghq.com/security/code-security/repositories
[2]: https://app.datadoghq.com/security/code-security/sca
[6]: /security/code_security/static_analysis/static_analysis_rules
```

#### Ignore for a specific instance

> To ignore a specific instance of a violation, comment `no-dd-sa` above the line of code to ignore. Rather than omitting the violation entirely, violations suppressed with `no-dd-sa` are shown as **suppressed** so that you can search and audit them.

Reviewer: ```suggestion
To ignore a specific instance of a violation, comment `no-dd-sa` above the line of code. Violations suppressed with `no-dd-sa` are shown as **suppressed**, rather than omitted entirely, so you can search and audit them.
```

> In the repository view, suppressed violations appear with `is_suppressed: true`. In the vulnerability explorer, they appear with `status: muted` and `workflow.mute.reason: muted_in_code`.

Reviewer: ```suggestion
On the [Repositories page][1], suppressed violations appear with `is_suppressed: true`. In the [Vulnerabilities explorer][2], they appear with `status: muted` and `workflow.mute.reason: muted_in_code`.
```

For example, in the following Python code snippet, the line `foo = 1` would be suppressed in Static Code Analysis scans.

```python
#no-dd-sa
foo = 1
bar = 2
```

> You can also use `no-dd-sa` to only suppress a particular rule rather than suppressing all rules. To do so, specify the name of the rule you wish to suppress in place of `<rule-name>` using this template:

Reviewer: ```suggestion
You can also use `no-dd-sa` to only suppress a particular rule, rather than suppressing all rules. To do so, specify the name of the rule you wish to suppress in place of `<rule-name>` using this template:
```

`no-dd-sa:<rule-name>`

> For example, in the following JavaScript code snippet, the line `my_foo = 1` is suppressed only for the `javascript-code-style/assignment-name` rule and is still analyzed by all other rules.

Reviewer: ```suggestion
For example, in the following JavaScript code snippet, the line `my_foo = 1` is suppressed only for the `javascript-code-style/assignment-name` rule, but all other rules still analyze it.
```
